cmd/waku/server/rest/message.go

```diff
@@ -0,0 +1,46 @@
+package rest
+
+import (
+	"errors"
+
+	"github.com/waku-org/go-waku/cmd/waku/server"
+	"github.com/waku-org/go-waku/waku/v2/protocol/pb"
+)
+
+type RestWakuMessage struct {
```

Comment:
Suggestion: Would be nice if we can generate this from yaml, but that can be taken up later once yamls get aligned across implementations.

Reply:
I agree! it should be possible to even generate the handlers for REST endpoints from the yaml. These are some projects i have looked that we could consider for this task:
- https://github.com/contiamo/openapi-generator-go (this one is cool because it also uses `chi`)
- https://github.com/ogen-go/ogen
- https://github.com/OpenAPITools/openapi-generator